### ISSUE TITLE
Train copy of non binary sequence

### ISSUE THREAD

**Issue by linamede** [Mostafa-Samir/DNC-tensorflow#3]
I would like to train a copy task for a non binary sequence. For example, for a sequence containing random numbers between [0, 1]. What loss function would you suggest? MSE? Iterations and lr should also be adjusted, right?

**Comment by Mostafa-Samir** [comment]
So you'll just have an input of size 1 and output of size 1 as well and both are real numbers? I believe MSE would be the choice here, and you'll need to remove the sigmoidal squashing of the output . However, and I can't put this into concrete reasons as I didn't think thoroughly about, but mostly due to precision problems I would be surprised if your model converged.
